db/migrate/20210208022738_move_new_since_to_new_table.rb

```diff
@@ -2,35 +2,40 @@
 
 class MoveNewSinceToNewTable < ActiveRecord::Migration[6.0]
   def up
-    sql = <<~SQL
+    User.find_in_batches do |users|
```

Comment:
I think we should always try to avoid ActiveRecord in migrations. Can we do this with raw sql? Maybe find the min and max user id, then group them in batches of 1000.